packages/client/bin/cli.ts

```diff
@@ -175,11 +175,14 @@ async function run() {
   }
 
   const common = new Common({ chain, hardfork: 'chainstart' })
+  const datadir = args.datadir ?? Config.DATADIR_DEFAULT
+  const key = await Config.getClientKey(datadir, common)
   const config = new Config({
     common,
     syncmode: args.syncmode,
     lightserv: args.lightserv,
-    datadir: args.datadir,
+    datadir,
+    key,
```

Comment:
I wonder though if we want to make the name of this option a bit more expressive?